docs/ar/week02/02.md

```diff
@@ -0,0 +1,52 @@
+
+---
+lang: ar
+lang-ref: ch.02
+title: أسبوع 2
+---
+
+<!--
+## Lecture part A
+
+We start by understanding what parametrised models are and then discuss what a loss function is. We then look at Gradient-based methods and how it's used in the backpropagation algorithm in a traditional neural network. We conclude this section by learning how to implement a neural network in PyTorch followed by a discussion on a more generalized form of backpropagation.
+-->
+
+## الجزء أ من المحاضرة
```

Comment:
Is it possible to use "A"?

Reply:
it will revert the Arabic string around the English one, it's going to be like:  of the lecture A part

Reply:
it will revert the Arabic string around the English one, it's going to be like:  of the lecture A part